reading of a file using fgets() .c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+main()
+{
+  char str[10];
+  file *fp;
+  fp = fopen("f1.txt"."r");
+  if(fp == null)
```

Comment:
Write "null" as "NULL" otherwise it shows the identifier as undefined.

Reply:
thanks for suggesting me....